### PR TITLE
Added method to API Controller, Utilizes Wiki API to add description …

### PR DESCRIPTION
**Pull request by procheteau**
- Add state_info method to ApplicationController, method uses Wiki API to collect info on states
- Updated SystemController to call new method
-Updated StateShowContainer to show state description

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -12,4 +13,43 @@ def configure_permitted_parameters
    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :home_state])
   end
 
+  base_uri "https://en.wikipedia.org/w/api.php?format=json&action=query&prop=extracts&exintro&explaintext&redirects=1&titles="
```
Comment (CassandraNewell) [inline app/controllers/application_controller.rb]: I think this is all in your parseWiki PORO now -- please remove from here!

File: app/controllers/states_controller.rb [hunk 2]
```diff
@@ -1,5 +1,13 @@
+require_relative "../models/parsewiki"
+
 class StatesController < ApplicationController
   def index
     @states = State.all
+    wiki = Parsewiki.new
+    binding.pry
```
[the remark this answers is not in the document]
Comment (CassandraNewell) [reply app/controllers/states_controller.rb]: and the one below it

File: db/seeds.rb [hunk 3]
```diff
@@ -10,7 +10,6 @@
   { :name => 'Colorado', :abbreviation => 'CO', :description => 'This is a state'},
   { :name => 'Connecticut', :abbreviation => 'CT', :description => 'This is a state'},
   { :name => 'Delaware', :abbreviation => 'DE', :description => 'This is a state'},
-  { :name => 'District of Columbia', :abbreviation => 'DC', :description => 'This is a state'},
```
Comment (CassandraNewell) [inline db/seeds.rb]: Taxation without representation! ✊

File: Gemfile [hunk 4]
```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```
Comment (CassandraNewell) [inline Gemfile]: Please also delete the file `.DS_Store`, which is a Mac artifact

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (adpaquin) [reply app/controllers/application_controller.rb]: We removed the parseWiki PORO and implemented it's functionality inside of the application_controller.rb as the new state_info method.

File: app/controllers/states_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (adpaquin) [reply app/controllers/states_controller.rb]: We removed file as it was not being used. We are using a controller in api/v1/sates_controller to get all of the states for our index page.

**Pull request merged**